eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -137,7 +139,7 @@ jobs:
       - pwsh: |
           ctest `
             -C Debug `
-            -V `
+            -VV `
```

Comment:
This is hardcoded.  Why do we want to use -VV ?

If the intention is to hardcode the value, then remove the `AZURE_LOG_LEVEL` parameters added above and to `jobs/archtype-sdk-tests.yml`.

If you want to make this configurable for each service, then keep the parameters and change this to ${{ parameters.AZURE_LOG_LEVEL }}


However, I haven't noticed a real difference between -V and -VV in the past.  Have you?  Is there something you are trying to catch?